modules/rules_doxygen/1.0.0/source.json

```diff
@@ -0,0 +1,5 @@
+{
+    "integrity": "sha256-L+PYwLJz7h3ji6y0C9GO9BS6w6ZTeHOzuyoaywjuwKk=",
+    "strip_prefix": "rules_doxygen-1.0.0",
+    "url": "https://github.com/TendTo/rules_doxygen/archive/refs/tags/1.0.0.tar.gz"
```

Comment:
Since you control the repo, could you add a release artifact? The `archive` does not guarantee a stable hash.